common.sh

```diff
@@ -9,6 +9,7 @@
 FUZZING_ENGINE=${FUZZING_ENGINE:-"libfuzzer"}
 [[ $FUZZING_ENGINE != "libfuzzer" ]] && [[ $FUZZING_ENGINE != "afl" ]] && echo "USAGE: If defined, $FUZZING_ENGINE should be either 'afl' or 'libfuzzer' but it is $FUZZING_ENGINE" && exit 1
 
+BINARY_NAME_EXT="_${FUZZING_ENGINE}"
```

Comment:
Indeed. I should not have mixed functionality for the main harness into the same commit.
Initially I was running the experiment from the top repo of the fuzzer-test-suite, thus the 
binary name was the same as the directory name, which is not a problem if one creates a dedicated dir. However it seems it would be useful if this is automatically addressed. Same  same applies to other requirements that are assumed (such as requirements for Fuzzer/ to already be present - which results in build_fuzzer automatically failing instead of automatically downloading libFuzzer from svn and continuing). If you think these are an issue feel free to open an issue on github and I'd be happy to contribute a patch.